src/cmsAdapter.js

```diff
@@ -0,0 +1,11 @@
+const getTenScreeningsAdapter = {
```

Comment:
The name of this variable confuses me, because it implies it's value should be an adapter for retrieving ten screenings, but the only function it holds is `loadAllScreenings()` which contradicts that. It's exported as the default export from a module named `cmsAdapter`, which further confuses things.